.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "21:00"
```

Comment:
Doesn't dependabot automatically work? The configuration file itself makes me nervous if its configured incorrectly.